plugins/kubernetes/app/models/kubernetes/cluster.rb

```diff
@@ -59,6 +59,13 @@ def schedulable_nodes
       []
     end
 
+    def server_version
+      version = Rails.cache.fetch(cache_key, expires_in: 1.hour) do
+        JSON.parse(client.create_rest_client('version').get.body).fetch('gitVersion')[1..-1]
```

Comment:
good call on not caching the `Gem::Version` ... custom ruby objects are just trouble :)